source/includes/_call_connect.md

```diff
@@ -0,0 +1,90 @@
+# Call Connect
+
+The Call Connect endpoint is used to create a phone call between a visitor and
+an advertiser. The advertiser's phone number is determined using the given visit
+ID.
+
+## Create
+
+### HTTP Request
+
+| Method | URI Format |
+|---|---|
+| POST | https://api.reachlocalservices.com/call_connect |
```

Comment:
Does this have to be a top level resource?  Could it be nested under capture_events so it would be 
```
https://api.reachlocalservices.com/capture_events/call_connect
```

Reply:
I don't really feel it would belong with capture_events... it doesn't serve a similar purpose. It's a mechanism to initialize a call. An event data structure will later be created to represent this, but this is the actual action.